dfusion_rust_core/Cargo.toml

```diff
@@ -7,12 +7,12 @@ edition = "2018"
 [dependencies]
 array-macro = "1.0.4"
 byteorder = "1.3.1"
-graph = { git = "https://github.com/graphprotocol/graph-node", tag = "v0.13.2" }
-graph-node-reader = { git = "https://github.com/gnosis/graph-node-reader", tag = "v0.13.2" }
+graph = { git = "https://github.com/graphprotocol/graph-node", tag = "v0.16.1" }
+graph-node-reader = { git = "https://github.com/gnosis/graph-node-reader", rev = "f77b37c" }
```

Comment:
let's tag it?

Reply:
OFC, just waiting to get e2e tests passing before I do, in cases there is something wrong with `graph-node-reader`. There aren't any tests there so I didn't feel comfortable with my changes.